(8) A conical tent is to have $1000$ft$^{3}$ capacity. Find its dimensions if the amount of canvas is to be a minimum. Disregard the floor. Note that the formula for the volume of a cone is $$V=\frac{1}{3}r^{2}\pi{}h$$ The formula for the area of the lateral surface of a cone (without the floor) is $$A=\pi{}rs$$ where $s$ is the slant height (see diagram).


<img src="coneslantheight.png">

**Step 1:** $A(r)=\pi{}rs$

**Step 2:** $s=\sqrt{r^{2}+h^{2}}$ and $\frac{1}{3}r^{2}\pi{}h=1000$ therefore $$s=\sqrt{r^{2}+\frac{9\cdot{}10^{6}}{r^{4}\pi^{2}}}$$ $$A(r)=\pi{}r\sqrt{r^{2}+\frac{9\cdot{}10^{6}}{r^{4}\pi^{2}}}=\pi{}\sqrt{r^{4}+\frac{9\cdot{}10^{6}}{r^{2}\pi^{2}}}$$

**Step 3:** $$A'(r)=\pi\cdot\frac{4r^{3}-\frac{18\cdot{}10^{6}}{r^{3}\pi^{2}}}{2\sqrt{r^{4}+\frac{9\cdot{}10^{6}}{r^{2}\pi^{2}}}}$$

It follows that, solving $A'(r)=0$, $$4r^{3}=\frac{18\cdot{}10^{6}}{r^{3}\pi^{2}}$$ Isolate $r$ for $$r=10\cdot\left(\frac{9}{2\pi^{2}}\right)^{\frac{1}{6}}$$

In [11]:
r=((9/(2*pi^2))^(1/6))*10

Since $h=\frac{3000}{r^{2}\pi}$

In [12]:
h=3000/(r^2*pi)

In [13]:
(n(r),n(h))

(8.77308077654739, 12.4070098179880)

Answer: if the amount of the canvas is to be at a minimum, then the dimensions should be approximately $r=8.77$ feet and $h=12.41$ feet.